internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4464,6 +4464,41 @@ func TestHandleGetInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetMempoolInfo(t *testing.T) {
+	t.Parallel()
+	txDescOne := &mempool.TxDesc{
+		StartingPriority: 0,
+	}
+	txDescOne.Tx = dcrutil.NewTx(block432100.Transactions[0])
```

Comment:
All of these could just be done as part of the definition to make it cleaner:

```Go
txDescOne := &mempool.TxDesc{
	StartingPriority: 0,
	Tx: dcrutil.NewTx(block432100.Transactions[0]),
	Type: stake.TxTypeSSGen,
	Fee: 1,
}
```

Reply:
Does it look any better now? It told me I couldn't assign to the promoted fields in a literal.

Reply:
Yep.  Forgot it was embedding the `mining.TxDesc`.  That update is proper.